File: Scripts/temp/classes/c-sharp/classes.cs
using System;

class MyClass{

  // instance variable
  private int variable;  // Note: instance variables are usually "private"

  /**
  * The constructor
  */
  public MyClass(){
    // creates a new instance
  }

  /**
  * A method
  */
  public void someMethod(){
   this.variable = 1;
    Console.WriteLine("Variable value is "+this.variable);
  }

public static void Main(String[] args){

for (int i = 0; i < 1000000; ++i) {
	MyClass obj = new MyClass();
	obj.someMethod();
}
}

}

File: Scripts/temp/classes/rust/classes.rs

fn main() {
    struct MyClass {
	variable: u32
  }

    impl MyClass {
        pub fn new() -> MyClass {
            MyClass { variable: 19022 }
        }
        pub fn someMethod(&self) -> u32 {
            return self.variable
        }
    }

for i in 0..1000000{
    let my_class = MyClass::new();
    println!("Variable value is {}", my_class.someMethod());
}
}
